Lib/platform.py

```diff
@@ -398,8 +398,8 @@ def win32_ver(release='', version='', csd='', ptype=''):
     else:
         try:
             cvkey = r'SOFTWARE\Microsoft\Windows NT\CurrentVersion'
-            with winreg.OpenKeyEx(HKEY_LOCAL_MACHINE, cvkey) as key:
-                ptype = QueryValueEx(key, 'CurrentType')[0]
+            with winreg.OpenKeyEx(winreg.HKEY_LOCAL_MACHINE, cvkey) as key:
+                ptype = winreg.QueryValueEx(key, 'CurrentType')[0]
         except:
```

Comment:
Maybe this bug was never seend because of "except: pass". It's maybe time to use more precise errors?

Reply:
Is `except OSError:` acceptable? Or are there any situations where this would raise an exception other than an `OSError`? And if so, is there a good reason why those are currently passing silently?

Reply:
win32_edition() doesn't have the typo and it uses "except OSError: pass", so yeah I think that "except OSError:" is reasonable here.